test/suites/smoke-test-dancelight/test-external-validators-rewards.ts

```diff
@@ -2,6 +2,8 @@ import "@tanssi/api-augment/dancelight";
 
 import { beforeAll, describeSuite, expect } from "@moonwall/cli";
 import type { ApiPromise } from "@polkadot/api";
+import { HOLESKY_SOVEREIGN_ACCOUNT_ADDRESS, PRIMARY_GOVERNANCE_CHANNEL_ID, getCurrentEraStartBlock } from "utils";
+import { getAccountBalance } from "../../utils/account.ts";
```

Comment:
Why 2 separate "utils"? Shouldn't this use the package syntax? As per https://github.com/moondance-labs/tanssi/pull/883

Reply:
Updated, thanks!